### PR TITLE
Clean up latest episodes on homepage

### PR DESCRIPTION
**Pull request by reesericci**
I removed pagination from Latest Episodes on the homepage, as it shouldn't be an ever-expanding list of episodes - that belongs to an "All Shows" page. Following in line with that, I shortened the amount of episodes shown on the homepage from 15 to 6.

Screenshot:

image

### CODE REVIEW

**Comment by graemearthur** [comment]
Think this would work with a more episodes link at the end for those who are looking for all episodes

**Comment by gerbrent** [comment]
Well, you see, this was specifically a feature request I made several weeks ago when the home page was instead the entire JB catalogue - to show many recent episodes, but not all, and w natural pagination.

#### My perspective and thus motivation for that ask is this:

* a new-to-JB listener will likely arrive at the first page, and be in the mood to explore. That can be done either by browsing the menu and sub-menus, or by scrolling. On mobile, scrolling down a page or two is a very intuitive action, and you thus get a good sense of JB's recent offering with more than 6 recent episodes on offer. 
* A seasoned JB listener will know there are several shows, and can either dig into the menu for the specific show they're after, or more often than not browse the last few episodes on offer. 
* If you are a casual JB listener (in contrast to a listen-to-everything type listener) and thus haven't listened in some time, exploring recent topics across all shows is a great way to re-engage said listener.
* with such a small recent-episodes offering on the front page, I get a small sense that perhaps this subset is the only offering on hand. It confuses the "Shows Carousel" with the "Recent Episodes" view, as they have the same artwork, similar shapes, similar number of items. The distinction between both feels obscured at that point in my opinion.

#### scenarios in which 6 or less is not ideal:

With only 6 recent episodes shown, there are regular sub-optimal scenarios where a show is not present on the home page, despite having a relatively recent episode:
* LAN251, Coder476, LUP468, Office8, LAN250, Coder475 - Where's the latest SSH?
* Office 9, LAN252, Coder477, LUP469, SSH76, LAN251 - Only one LUP present, our biggest show?

#### Alternatives
I would welcome discussion and perspective around alternatives - a `more` button might just be the right touch - however I do not believe the above commits and sample without pagination or `more` button is the answer.

Another consideration: A `more` button would thus require an "All Shows" page, which is discussed and strategy not yet decided in #162

**Comment by gerbrent** [comment]
In conclusion, I do not feel this should be merged until more discussion can happen and a decision be made on approach.

**Comment by reesericci** [comment]
Could we increase the count of episodes from 6 to 9?

—reese

On August 18, 2022, Bananaman ***@***.***> wrote:
> In my opinion this should not be merged for the reasons you stated
> above @gerbrent <https://github.com/gerbrent>. Tbh I'd close this PR
> too as noone else seems to have strong opinions in the opposite
> direction.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/236#issuecomment-
1219419301>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AEVOPDVTTVRRDAZKN6XVOI3VZYSOJANCNFSM55ZGYP6A>.
You are receiving this because you authored the thread.Message ID:
***@***.***>

**Comment by gerbrent** [comment]
@reesericci: @ChrisLAS was interested in the same, though i think we should keep pagination...

**Comment by reesericci** [comment]
I think pagination _is_ hurting the design because "Latest Episodes" shouldn't be an ever-flowing stream of content & makes the design less elegant imo.

**Comment by ironicbadger** [comment]
Your opinion is noted. Thank you!

**Comment by reesericci** [comment]
Is that supposed to be passive-agressive?

—reese

On August 18, 2022, Bananaman ***@***.***> wrote:
> Your opinion is noted. Thank you!
>

—
Reply to this email directly, view it on GitHub
<https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/236#issuecomment-
1219811102>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AEVOPDSQ6X3VCXK4MIUOG4DVZZ7EVANCNFSM55ZGYP6A>.
You are receiving this because you were mentioned.Message ID:
***@***.***>

**Comment by gerbrent** [comment]
> I think pagination _is_ hurting the design because "Latest Episodes" shouldn't be an ever-flowing stream of content & makes the design less elegant imo.

I feel like a "more" button that links to a (currently non-existent) `All Episodes, sorted by latest` page would be a really happy solution to this (and potentially to partially #162 aswell). Though it can be said that it's a small detail for Milestone 1.0, and our collective efforts are likely better spent elsewhere on more pressing and essential tasks.

**Comment by gerbrent** [comment]
https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/236#issuecomment-1207317664

see here also as a reminder of all my particular reasoning behind the above.

**Comment by ironicbadger** [comment]
> Is that supposed to be passive-agressive? —reese

Not at all :) Just wanted to let you know that you've been heard even if we don't go with your suggestion.

**Comment by reesericci** [comment]
Just changed from 15 to 9 and resolved merge conflicts!

File: themes/jb/layouts/index.html [hunk 1]
```diff
@@ -62,9 +63,6 @@ <h1 class="title">Latest Episodes</h1>
 
       </div>
     </div>
-    <div class="container">
-          {{ partial "helper/pagination.html" . }}
-    </div>
```
Comment (gerbrent) [inline themes/jb/layouts/index.html:66]: We would like to keep the pagination. Could you keep this section please?

Comment (gerbrent) [reply themes/jb/layouts/index.html:66]:
```suggestion
    <div class="container">
          {{ partial "helper/pagination.html" . }}
    </div>
```

**Review by gerbrent — CHANGES_REQUESTED** [review]

**Comment by reesericci** [comment]
Ok just implement your changes!

**Review by gerbrent — APPROVED** [review]

**Review by gerbrent — APPROVED** [review]
Tested and all is perfect! Thanks @reesericci !

**Pull request merged**